### PR TITLE
Use github actions instead of travis ci for faster builds

### PR DESCRIPTION
**Pull request by mmachatschek**
[Actions](https://github.com/pusher/pusher-http-php/settings/actions) need to be activated

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          extensions: curl, sodium
```
Comment (GrahamCampbell) [inline .github/workflows/test.yml:24]: Don't add `curl` here. it is already enabled by default.

File: composer.json [hunk 2]
```diff
@@ -23,11 +23,6 @@
     "config": {
         "preferred-install": "dist"
     },
-    "extra": {
-        "branch-alias": {
-            "dev-master": "3.4-dev"
```
Comment (GrahamCampbell) [inline composer.json:28]: Why is this removed?

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: composer.json (hunk 2 above)
Comment (mmachatschek) [reply composer.json]: Fixes. Was a local Charger i forgot about

File: .github/workflows/test.yml (hunk 1 above)
Comment (mmachatschek) [reply .github/workflows/test.yml:24]: Why not adding it as a dependency as it‘s a requirement of the library? Its not breaking anything and ensures it‘s installed if something changes in the php installer.

This is also added in other big packages like laravel‘s, it‘s not required their too, but still explicitly added in the workflows

**Review by elverkilde — APPROVED** [review]
